tests/vmi_multus_test.go

```diff
@@ -882,15 +883,27 @@ var _ = Describe("[Serial]SRIOV", func() {
 })
 
 var _ = Describe("[Serial]Macvtap", func() {
-	virtClient, err := kubecli.GetKubevirtClient()
-	tests.PanicOnError(err)
-	macvtapLowerDevice := "eth0"
-	macvtapNetworkName := "net1"
+	var err error
+	var virtClient kubecli.KubevirtClient
+	var macvtapLowerDevice string
+	var macvtapNetworkName string
 
 	tests.BeforeAll(func() {
+		virtClient, err = kubecli.GetKubevirtClient()
+		tests.PanicOnError(err)
+
+		macvtapLowerDevice = "eth0"
+		macvtapNetworkName = "net1"
+
 		// cleanup the environment
 		tests.BeforeTestCleanup()
+	})
+
+	BeforeEach(func() {
```

Comment:
Because if I fail to disable the feature gate I still want to be remove the network attachment definition; failing to delete a part of a test should not be a reason to skip deletion of the remaining entities.

Same for pretty much everything that will have multiple sections.